benchmark/wildtime_benchmarks/example_pipelines/yearbook.yaml

```diff
@@ -0,0 +1,50 @@
+pipeline:
+  name: Yearbook Test Pipeline
+  description: Example pipeline
+  version: 1.0.0
+model:
+  id: YearbookNet
+  config:
+    num_input_channels: 1
+    num_classes: 2
+training:
+  gpus: 1
+  device: "cpu"
```

Comment:
I guess that should be consistent over all pipelines

Reply:
Yep, I think it remained CPU because it is the only one you can run locally without problems